### ISSUE TITLE
Hide disabled tools in the Classifier toolbar

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/front-end-monorepo#732]
**Package**

lib-classifier

**Feature or Issue Description**

From the beta feedback, we should hide the unused toolbar buttons rather than disable them.

cc #562

**Comment by mcbouslog** [comment]
Currently from [ImageToolbar](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/ImageToolbar/ImageToolbar.js#L39-L40) a `disabled` boolean prop is passed down to [the Button component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js) where the magic happens.

Should I remove code related to disabling the buttons?

To hide the Rotate and Fullscreen buttons I'm thinking to add a different prop like `hidden` to conditionally return null (watch out, 4 line PR incoming!) or hide with CSS? 

If hide with CSS, could do `visibility: hidden`, `display: none` or other option. Since the buttons' container uses `flex`, I think `visibility` or `display` has no visual difference, and both are hidden from screen readers (which I think we want), so not sure if we prefer one over the other.

I'm leaning towards just returning null if there's a `hidden` prop. And if we've decided these image toolbar buttons shouldn't show as disabled, I might as well remove the related code. Returning null would be easily removed once we've implemented the buttons.
